docs/fsm/graphviz/draw_state_diagrams.py

```diff
@@ -1,3 +1,11 @@
+import os
+import sys
+
+sys.path.insert(
+    0, os.path.abspath(os.path.join(os.path.dirname(__file__), "../../../src"))
```

Comment:
You walked the 3 folders up before you walk them down again? I remember you saying something about that "being hackish"? :D

Reply:
lol good point :). Resolved.